public/css/icinga/base.less

```diff
@@ -151,9 +150,55 @@
     --gray-light: #d0d3da;
     --gray-lighter: #e8ecef;
     --gray-lightest: #F7F7F7;
+
+    // ipl-web overrides
+    --base-gray: var(--gray);
+    --base-gray-semilight: var(--gray-semilight);
+    --base-gray-light: var(--gray-light);
+    --base-gray-lighter: var(--gray-light);
+
+    --base-remove-bg: @color-critical;
```

Comment:
This will be initialized by ipl-web with `@state-critical`, so it's redundant then and I'll remove it.